### ISSUE TITLE
kubeflow 1.3 not creating gpu notebooks

### ISSUE THREAD

**Issue by brettkoonce** [canonical/bundle-kubeflow#359]
I was able to get a kubeflow 1.3 installation using microk8s 1.21 and the steps documented here (with a --purge of microk8s/juju before):  https://github.com/ubuntu/microk8s/issues/2253#issuecomment-838158746

However, when creating a new notebook, I get the following error:

<img width="685" alt="Screen Shot 2021-05-11 at 11 40 03 AM" src="https://user-images.githubusercontent.com/11281814/118003094-dd14b280-b30d-11eb-8c55-060b96d90fec.png">

Vanilla config, eg:

<img width="920" alt="Screen Shot 2021-05-11 at 11 40 15 AM" src="https://user-images.githubusercontent.com/11281814/118003134-e867de00-b30d-11eb-87e1-5ae0a92e8cf1.png">

**Comment by DomFleischmann** [comment]
Thank you for pointing this out! Have you had any success with a custom image?

**Comment by DomFleischmann** [comment]
Also is this with the microk8s gpu addon enabled?

**Comment by brettkoonce** [comment]
yes I am using the GPU addon, it pulled down the new operator I believe.  Will try a custom image in a bit.

**Comment by brettkoonce** [comment]
hmm...

> microk8s.kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"
> NAME     GPU
> athena   none

**Comment by DomFleischmann** [comment]
So it might be an issue with `microk8s.enable gpu` @joedborg or @ktsakalozos, is this output expected?

**Comment by brettkoonce** [comment]
creating cpu-only containers works, for what it's worth.  happy to tear down/rebuild things once the official release goes live.  happy to help trouble shoot things!

**Comment by aencalado** [comment]
Any updates? I am experiencing the same problem

**Comment by brettkoonce** [comment]
still seeing this w/ 20.04 + microk8s 1.20 + 

```
sudo snap install microk8s --classic
microk8s.enable gpu
microk8s enable kubeflow --bundle=cs:kubeflow
```
